cmd/native-relayer/cli/start.go

```diff
@@ -0,0 +1,147 @@
+package cli
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	"github.com/block-vision/sui-go-sdk/signer"
+	"github.com/block-vision/sui-go-sdk/sui"
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/gonative-cc/relayer/dal"
+	"github.com/gonative-cc/relayer/env"
+	"github.com/gonative-cc/relayer/ika"
+	"github.com/gonative-cc/relayer/ika2btc"
+	"github.com/gonative-cc/relayer/native"
+	"github.com/gonative-cc/relayer/native2ika"
+	"github.com/gonative-cc/relayer/nbtc"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+var startCmd = &cobra.Command{
+	Use:   "start",
+	Short: "Starts the relayer",
+	Run: func(cmd *cobra.Command, args []string) {
+		lvl, err := cmd.Root().PersistentFlags().GetString("log-level")
+		if err != nil {
+			log.Error().Err(err).Msg("Error getting log level")
+			os.Exit(1)
+		}
+		logLvl, err := zerolog.ParseLevel(lvl)
+		if err != nil {
+			log.Error().Err(err).Msg("Error parsing log level")
+			os.Exit(1)
+		}
+		env.InitLogger(logLvl)
+		configFile, err := cmd.Root().PersistentFlags().GetString("config")
+		if err != nil {
+			log.Error().Err(err).Msg("Error getting config file path")
+			os.Exit(1)
+		}
+
+		config, err := loadConfig(configFile)
+		if err != nil {
+			log.Error().Err(err).Msg("Error loading config")
+			os.Exit(1)
+		}
```

Comment:
let's move lines 29 (from the beginning) -59 (loading config) to a separate helper function, eg:
```
config := prepareEnv(flags)
```

Reply:
Done